core/src/main/java/org/elasticsearch/monitor/fs/FsProbe.java

```diff
@@ -135,14 +135,22 @@ public FsInfo stats(FsInfo previous, @Nullable ClusterInfo clusterInfo) throws I
         return Files.readAllLines(PathUtils.get("/proc/diskstats"));
     }
 
+    /** See: https://bugs.openjdk.java.net/browse/JDK-8162520 */
```

Comment:
Sorry to be a pain, but can you make this not a Javadoc, just a regular comment? 😦